src/ProxyInterfaceSourceGenerator/FileGenerators/ExtraFilesGenerator.cs

```diff
@@ -27,6 +27,7 @@ internal sealed class ProxyAttribute : Attribute
         public Type Type {{ get; }}
         public bool ProxyBaseClasses {{ get; }}
         public ProxyClassAccessibility Accessibility {{ get; }}
+        public string[] MembersToIgnore {{ get; }}
```

Comment:
I think that nullable parameters are not supported for an attribute?

`public string[]? MembersToIgnore {{ get; }}`